static/src/stylesheets/module/site-messages/_first-pv-consent.scss

```diff
@@ -28,7 +28,32 @@ $btn-height: ($gs-baseline*3.5);
     }
 
     .site-message__roundel {
+        position: absolute;
+        padding: 0;
+        width: auto;
+        height: auto;
         display: none;
+
+        .inline-icon {
+            fill: $brightness-100;
+            path:last-child {
+                fill: $brightness-20;
+            }
+        }
+
+        svg {
+            width: 36px;
```

Comment:
can we make the roundel size variable global? seems to be consistent at least across banners

Reply:
any idea how/where we should set that?

Reply:
either way probs best to make a dedicated PR for that, and use it in the action required banner as you suggested on that PR